src/ImageManipulation.php

```diff
@@ -46,6 +46,9 @@
  */
 trait ImageManipulation
 {
+    use Attributable;
```

Comment:
trait in a trait :-p

Reply:
Inception! I don't think there's anything inherently wrong with it and we actually have a couple other places where we do this already:
- Extensible
- FileUploadReceiver